prime-router/src/main/kotlin/azure/TokenFunction.kt

```diff
@@ -41,9 +48,21 @@ class TokenFunction(val metadata: Metadata = Metadata.getInstance()) : Logging {
         // &grant_type=client_credentials
         // &client_assertion_type=urn:ietf:params:oauth:client-assertion-type:jwt-bearer
         // &client_assertion=verylong.signed.jwtstring
-        val clientAssertion = request.queryParameters["client_assertion"]
+        //
```

Comment:
Maybe mention in the comments that `Content-Type:` should be `'application/x-www-form-urlencoded; charset=utf-8'` or `application/x-www-form-urlencoded` and that this format of sending parameters in the body is how form data is POSTed to a server?

Reply:
Done.